### PR TITLE
Add PSTAKE

### PR DESCRIPTION
**Pull request by ChipKlutzy**
### Checklist
- [x] Token Address in lowercase
- [x] Link to verified contract on Blastscan
- [x] Description of token if non-standard ERC20, e.g. has tax, OFT token, bridged XERC20, ERC-404, etc.
- [x] An SVG <150 KB in size, cropped to be a circle. **Rectangles and other shapes of images will not be accepted.**
- [x] Token category you would like to be listed under
    - Current categories: DeFi, Memecoin, Stablecoin, Other

### Additional Information
-

### CODE REVIEW

File: chains/81457/token-list.json [hunk 1]
```diff
@@ -384,6 +384,13 @@
     "tokenDecimals": 18,
     "tokenCategory": "DeFi"
   },
+  {
+    "tokenAddress": "0xd9a1266a23ed88fc9cb12454a5d6d338b65a3302",
```
Comment (0xjazper) [inline chains/81457/token-list.json]: u did not run the script to autogenerate this file, it should be alphabetically ordered

**Review by 0xjazper — CHANGES_REQUESTED** [review]
as per comment

File: chains/81457/token-list.json (hunk 1 above)
Comment (ChipKlutzy) [reply chains/81457/token-list.json]: Yeah the problem is when ever I run the script (./update_list.sh), it gives me blank output in "token-list.json" file.
Refer this [commit](https://github.com/ThrusterX/thruster-token-list/pull/24/commits/88ec868c324819c243f0f27b686c58b554983f66).

Comment (0xjazper) [reply chains/81457/token-list.json]: are you running the script from the root directory?

**Comment by 0xjazper** [comment]
i went ahead and added a pr for you here: https://github.com/ThrusterX/thruster-token-list/pull/30

will leave this pr up for a day or two before closing if you have any additional questions or concerns, otherwise, the token should be added for you

**Pull request closed without merging**